hack/make/tests/e2e.mk

```diff
@@ -1,107 +1,108 @@
 ## Start a test and skip TEARDOWN steps if it fails
```

Comment:
Please add a make target for edgeconnect test

Reply:
done